core/cog-webkit-utils.c

```diff
@@ -72,13 +72,6 @@ cog_handle_web_view_load_failed (WebKitWebView  *web_view,
                                  GError         *error,
                                  void           *userdata)
 {
-    // If the resource is going to be shown by a plug-in (or a
-    // media engine) just return FALSE and let WebKit handle it.
-    if (g_error_matches (error,
-                         WEBKIT_PLUGIN_ERROR,
-                         WEBKIT_PLUGIN_ERROR_WILL_HANDLE_LOAD))
-        return FALSE;
-
```

Comment:
Couldn't we use conditional compilation here to enable this part of the code only when building against the old API?

Reply:
We know that if API<2.0, then this code needs to be built.

Reply:
It's not just deprecated API, the functionality was removed, so it's dead code.